packages/rspack/src/logging/createConsoleLogger.ts

```diff
@@ -81,25 +47,14 @@ const LogLevel = {
 	verbose: 1
 };
 
-/**
- * @param {LoggerOptions} options options object
- * @returns {function(string, LogTypeEnum, any[]): void} logging function
- */
-export = ({ level = "info", debug = false, console }: any) => {
+export = ({ level = "info", debug = false, console }: LoggerOptions) => {
```

Comment:
I think it is okay for you to migrate it to
`export { createConsoleLogger }`
you can have a try